cscore/src/main/java/edu/wpi/first/cscore/VideoMode.java

```diff
@@ -75,4 +77,29 @@ public VideoMode(PixelFormat pixelFormat, int width, int height, int fps) {
 
   /** Frames per second. */
   public int fps;
+
+  @Override
+  public boolean equals(Object other) {
+    if (this == other) {
+      return true;
+    }
+    if (other == null) {
+      return false;
+    }
+    if (getClass() != other.getClass()) {
+      return false;
+    }
+    VideoMode mode = (VideoMode) other;
+    if (pixelFormat != mode.pixelFormat) return false;
+    if (width != mode.width) return false;
+    if (height != mode.height) return false;
+    if (fps != mode.fps) return false;
```

Comment:
Format is complaining that these don't have brackets

You could and the equality checks together and return the result